notes/v1.5.5.md

```diff
@@ -0,0 +1,146 @@
+# bloop `v1.5.5`
+
+Bloop v1.5.5 is a bugfix release.
```

Comment:
Bloop v1.5.5 is a bugfix release, but it also contains a major version bump of the `scala-debug-adapter` with [many new features](https://github.com/scalacenter/scala-debug-adapter/releases/tag/v3.0.1).

So I would appreciate some advertisement of that.

> Bloop v1.5.5 ships a new major version of the Scala Debug Adapter, featuring better debugging steps (step filter), conditional breakpoint, logpoints and more. See a full lists of features in the releases notes of the Scala Debug Adapter [3.0.1](https://github.com/scalacenter/scala-debug-adapter/releases/tag/v3.0.1) and [3.0.3](https://github.com/scalacenter/scala-debug-adapter/releases/tag/v3.0.3).

Reply:
If the classpath of projects depending on bloop artifacts now contains the new version of the debugger, it would be better to also bump the major version of bloop (since the new classpath is not binary compatible with the old classpath).

Reply:
Bloop frontend is a tool not a library, so I don't think we need to follow the semantic versioning rules. Scala Debug Adapter uses `VersionScheme.Strict` which means that we can break the binary compatibility between any two versions, even a patch version. As a consequence, on Bloop, it can also break the binary compatibility between any two patch versions.

Reply:
Let's do that in a follow up then.

>So I would appreciate some advertisement of that. Something like the following:

Added! Though I think it will be better visible within Metals, which I also plan to add.